pyproject.toml

```diff
@@ -1,6 +1,41 @@
+[project]
+name = "open-mss"
+version = "10.1.0"
+description = "MSS - Mission Support System"
+readme = "README.md"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+]
+keywords = ["mslib"]
+maintainers = [
+    {name = "Reimar Bauer", email = "rb.proj@gmail.com"},
+]
+authors = [
+    {name = "Marc Rautenhaus", email = "wxmetvis@posteo.de"},
```

Comment:
We maybe can add here all from the AUTHORS file?

Reply:
That would make sense, but I initially didn't want to push too much into this PR. If me put them all here and removed the AUTHORS file entirely we have to update all copyright notices in every file and update some things in the docs.